malduck/pe.py

```diff
@@ -34,27 +34,27 @@ def set_bytes_at_offset(self, offset, data):
         return True
 
 
-class MemoryPEData(object):
+class MemoryPEData:
     """
     `pefile.PE.__data__` represents image file usually aligned to 512 bytes.
     MemoryPEData perform mapping from pefile's offset-access to Memory object va-access
     based on section layout.
     """
 
-    def __init__(self, memory, fast_load):
+    def __init__(self, memory, fast_load) -> None:
```

Comment:
seems to be missing types for `memory` and `fast_load`?

Reply:
Fixed, complete PE typings will be added in separate PR